src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -140,68 +140,67 @@ private void startInactiveConsumerDeletionTimer(Long timeout) {
 
     @Override
     public void start(Promise<Void> startPromise) {
-
         RouterBuilder.create(vertx, "openapi.json")
-                .onComplete(ar -> {
```

Comment:
please don't change this indentation. It has been working pretty fine with people involved in the bridge so far. It's also not related to the PR.

Reply:
That was unintentional, and due to conflicts resolution. Reverted.